bin/live/pycbc_live_combine_single_significance_fits

```diff
@@ -93,19 +97,20 @@ for f in args.trfits_files:
             )
             continue
 
-        # We now determine the (approximate) start/end times of the
-        # trigger_fits file via the time of the first/last triggers in it.
-        # Ideally this would be recorded exactly in the file.
-        gps_last = 0
-        gps_first = np.inf
-        for ifo in args.ifos:
-            if ifo not in fits_f:
-                continue
-            trig_times = fits_f[ifo]['triggers']['end_time'][:]
-            gps_last = max(gps_last, trig_times.max())
-            gps_first = min(gps_first, trig_times.min())
-        trigger_file_starts.append(gps_first)
-        trigger_file_ends.append(gps_last)
+        # Get the start/end times of the trigger_fits file
+        fit_start = fits_f.attrs['fit_start']
+        fit_end = fits_f.attrs['fit_end']
```

Comment:
I suggest renaming `fit_start` / `fit_end` to `fit_start_time` / `fit_end_time`, at least in the HDF file, so as to have HDF keys be as self-documenting as possible.